src/main/groovy/io/openliberty/tools/gradle/tasks/InstallLibertyTask.groovy

```diff
@@ -59,25 +59,57 @@ class InstallLibertyTask extends AbstractTask {
         return project.configurations.libertyRuntime
     }
 
+    @Input
+    @Optional
+    String getLibertyRuntimeUrl() {
+        return project.liberty.install.runtimeUrl
+    }
```

Comment:
What about the project.liberty.runtime properties?
<img width="914" alt="Screen Shot 2021-02-26 at 3 17 17 PM" src="https://user-images.githubusercontent.com/29436451/109356011-bc2bdd80-7845-11eb-8142-46336fbf3f6d.png">

Reply:
That is in the liberty extension. We also support gradle properties for those.
<img width="1030" alt="Screen Shot 2021-02-26 at 3 21 01 PM" src="https://user-images.githubusercontent.com/29436451/109356408-470cd800-7846-11eb-811e-ca7d86560349.png">